provd/internal/services/testdata/TestRegisterGRPCServices/golden

```diff
@@ -126,10 +126,10 @@ privacy.PrivacyService:
     metadata: protos/privacy.proto
 timezone.TimezoneService:
     methods:
-        - name: SetTimezone
+        - name: GetTimezone
```

Comment:
hum, I am puzzled about the order (and the reordering of all those calls), are the golden file reproductible, always printing the same state order everytime?

Reply:
No, they actually update the order every time I see.

Here are two outputs of running with `--update`, one after the other

```
diff --git a/provd/internal/services/testdata/TestRegisterGRPCServices/golden b/provd/internal/services/testdata/TestRegisterGRPCServices/golden
index 80ed5885..cf3b1da0 100644
--- a/provd/internal/services/testdata/TestRegisterGRPCServices/golden
+++ b/provd/internal/services/testdata/TestRegisterGRPCServices/golden
@@ -1,107 +1,107 @@
 accessibility.AccessibilityService:
     methods:
-        - name: GetScreenKeyboard
+        - name: GetVisualAlerts
           isclientstream: false
           isserverstream: false
-        - name: GetStickyKeys
+        - name: EnableVisualAlerts
           isclientstream: false
           isserverstream: false 
...
```

```
diff --git a/provd/internal/services/testdata/TestRegisterGRPCServices/golden b/provd/internal/services/testdata/TestRegisterGRPCServices/golden
index 80ed5885..bb1c558c 100644
--- a/provd/internal/services/testdata/TestRegisterGRPCServices/golden
+++ b/provd/internal/services/testdata/TestRegisterGRPCServices/golden
@@ -1,107 +1,107 @@
 accessibility.AccessibilityService:
     methods:
-        - name: GetScreenKeyboard
+        - name: GetHighContrast
           isclientstream: false
           isserverstream: false
-        - name: GetStickyKeys
+        - name: GetSlowKeys
           isclientstream: false
           isserverstream: false
...
```

Reply:
Yeah, so the order is not deterministic. It’s definitively something that has to be fixed/figured out before merging this, or you will go crazy on every update :)

As we calling them in a non deterministic order in those tests? Is it something expected and only what matters is what get called in terms of how many times they are called with each parameters?
If so, we can have some reorder function that we intercept before saving in the golden file.